src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -949,11 +949,27 @@ public SamePathOrUnderConstraint SamePathOrUnder(string expected)
         /// Returns a constraint that tests whether the actual value falls 
         /// within a specified range.
         /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
         public RangeConstraint InRange(IComparable from, IComparable to)
```

Comment:
Why can't these two methods be `(object from, object to, IComparer comparer = null)` with `comparer` defaulting to `Comparer<object>.Default`? That's more what I expect from a .NET library.